src/mn_setup.c

```diff
@@ -5090,4 +5124,5 @@ void MN_BindMenuVariables(void)
     BIND_BOOL_MENU(csmenu.doubleammo);
     BIND_BOOL_MENU(csmenu.halfplayerdamage);
     BIND_BOOL_MENU(csmenu.aggromonsters);
+    BIND_NUM_MENU(menu_freelook, FREELOOK_OFF, FREELOOK_DIRECT_AIM);
```

Comment:
The idea is to use a non-config, menu-only variable. It's a feature that I added to the "custom skill" PR.